app/api/entities/free_scan_entity.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Entities
```

Comment:
This Entity is based on an outdated version of the ContainerEntity. It's code is needlessly complicated and should be recreated in a way that makes the container structure for scan containers understandable.